pkg/reconciler/subscription/controller.go

```diff
@@ -45,15 +48,24 @@ func NewController(
 
 	subscriptionInformer := subscription.Get(ctx)
 	channelInformer := channel.Get(ctx)
+	serviceaccountInformer := serviceaccountinformer.Get(ctx)
 
-	featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"))
+	var globalResync func(obj interface{})
+
+	featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"), func(name string, value interface{}) {
+		if globalResync != nil {
```

Comment:
`globalResync` seems always to be nil, as it is not set anywhere.
You have to set it e.g. via 
```
globalResync = func(obj interface{}) {
	impl.GlobalResync(subscriptionInformer.Informer())
}
```